src/VisualStudio/Core/Def/ProjectSystem/CPS/IWorkspaceProjectContextFactory.cs

```diff
@@ -70,17 +75,29 @@ internal abstract class EvaluationData
         /// <returns>
         /// Returns empty string if the property is not set.
         /// </returns>
-        /// <exception cref="ArgumentException">
+        /// <exception cref="InvalidProjectDataException">
         /// The <paramref name="name"/> is not listed in <see cref="IWorkspaceProjectContextFactory.EvaluationPropertyNames"/>
         /// </exception>
         public abstract string GetPropertyValue(string name);
 
+        /// <summary>
+        /// Returns the values of items of the specified <paramref name="name"/>.
+        /// </summary>
+        /// <returns>
+        /// Returns empty array if the items are not set.
+        /// </returns>
+        /// <exception cref="InvalidProjectDataException">
+        /// The <paramref name="name"/> is not listed in <see cref="IWorkspaceProjectContextFactory.EvaluationItemNames"/>
+        /// </exception>
+        public virtual ImmutableArray<string> GetItemValues(string name)
+            => ImmutableArray<string>.Empty;
```

Comment:
Needs to be virtual to avoid breaking project system, which implements this class.

Reply:
I'll implement this soon, then we can make it abstract if preferred.